pydantic_core/core_schema.py

```diff
@@ -170,6 +175,25 @@ class FunctionPlainSerSchema(TypedDict, total=False):
     function: Required[SerializePlainFunction]
     json_return_type: JsonReturnTypes
     when_used: WhenUsed  # default: 'always'
+    needs_model_instance: bool
+
+
+def _function_plain_ser_schema(
+    function: SerializePlainFunction | SerializePlainFunctionInstanceMethod,
+    json_return_type: JsonReturnTypes | None,
+    when_used: WhenUsed,
+    needs_model_instance: bool,
```

Comment:
I've iterated through several. None seem great because I think fundamentally this is the wrong sorta abstraction (we should be calling the method on the instance or something so that the caller doesn't care). But I'm good with whatever, we should just make this and #439 match